builder/builder.go

```diff
@@ -3,13 +3,24 @@ package builder
 import (
 	"context"
 	"fmt"
+
 	"github.com/kiegroup/container-builder/api"
 	"github.com/kiegroup/container-builder/client"
 	"github.com/kiegroup/container-builder/util"
 	"github.com/kiegroup/container-builder/util/log"
 	corev1 "k8s.io/api/core/v1"
 )
 
+type BuilderProperty string
+
+const KanikoCache BuilderProperty = "kaniko-cache"
+
+type BuilderInfo struct {
+	FinalImageName  string
+	BuildUniqueName string
```

Comment:
can this value be generated and saved into context?

Reply:
No, it's based on the client code's request.

Reply:
I see, thanks.